packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -49,32 +51,149 @@ function getSender(
   return isSMSAction(step.action) ? step.sender || SENDER_ID : step.senderName || SENDER_NAME;
 }
 
+async function isAuthorized(
```

Comment:
user either needs to own the  workflow or must be part of the team the workflow belongs to